scripts/Install-VSCode.ps1

```diff
@@ -132,7 +134,7 @@ param(
     [string]$Architecture = "64-bit",
 
     [parameter()]
-    [ValidateSet("Stable", "Insider-System", "Insider-User")]
+    [ValidateSet("Stable","Stable-User", "Insider-System", "Insider-User")]
```

Comment:
Since you're adding `Stable-User` can you update `Stable` to be `Stable-System` for consistency?

Reply:
Done in 544a208c0981887fe1a26a12ab04642b5647a8af